### PR TITLE
fix show_tag_keys in cluster mode

### PR DESCRIPTION
**Pull request by CrazyJvm**
fix issue: https://github.com/influxdb/influxdb/issues/3295

### CODE REVIEW

**Comment by CrazyJvm** [comment]
- [x] CHANGELOG.md updated
- [x] Rebased/mergable
- [x] Tests pass
- [x] Sign [CLA](http://influxdb.com/community/cla.html)

**Comment by CrazyJvm** [comment]
@otoolep  please help to review

**Comment by otoolep** [comment]
cc @dgnorton

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ With this release InfluxDB is moving to Go 1.5.
 - [#4622](https://github.com/influxdb/influxdb/issues/4622): Fix panic when passing too large of timestamps to OpenTSDB input.
 - [#5064](https://github.com/influxdb/influxdb/pull/5064): Full support for parenthesis in SELECT clause, fixes [#5054](https://github.com/influxdb/influxdb/issues/5054). Thanks @mengjinglei
 - [#5079](https://github.com/influxdb/influxdb/pull/5079): Ensure tsm WAL encoding buffer can handle large batches.
+- [#5101](https://github.com/influxdb/influxdb/pull/5101): Fix `show tag keys` in cluster mode
```
Comment (otoolep) [inline CHANGELOG.md]: Please add "Thanks @CrazyJvm " to this line!

**Comment by CrazyJvm** [comment]
Any feedback to this PR?

**Comment by otoolep** [comment]
Bump @dgnorton

**Comment by dgnorton** [comment]
@CrazyJvm  can you add integration test(s) that fail without this fix and pass with it?  The test(s) would go in `cmd/influxd/run`.  It's possible that you just need to add it to `server_suite_test.go`.  Thanks!

**Comment by krogon-intel** [comment]
I still got failure with this fix
`2016/01/27 15:53:57 failure getting snapshot from 10.0.0.2:8088: Get http://10.0.0.2:8088?index=0: read tcp 10.0.0.1:47388->10.0.0.2:8088: read: connection reset by peer`

**Comment by jwilder** [comment]
#5196 changed the query engine substantially so this PR is not able to be merged as is.  We are still fixing remote query execution due to the changes in #5196, but when that is complete, `show tag keys` should work.  We're tracking the cluster meta-queries in #5581.

**Pull request closed without merging**